src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -261,6 +261,15 @@
   <data name="SetViewStateUserKeyMessage" xml:space="preserve">
     <value>The subclass {0} of Page does not set the ViewStateUserKey property in method OnInit</value>
   </data>
+  <data name="UseContainerLevelAccessPolicy" xml:space="preserve">
+    <value>Use Container Level Access Policy</value>
+  </data>
+  <data name="UseContainerLevelAccessPolicyDescription" xml:space="preserve">
+    <value>No access policy identifier is used which makes it a non-revocable token limiting the SAS access to only the request customer's id. To grant longer-term permissions or to retain the ability to modify or revoke permissions after handing URL out, it's better to use container-level access policies.</value>
+  </data>
+  <data name="UseContainerLevelAccessPolicyMessage" xml:space="preserve">
+    <value>Use container-level access policy when creating Shared Access Signature (SAS)</value>
```

Comment:
>Use container-level access policy when creating Shared Access Signature (SAS) [](start = 11, length = 77)

Consider using Azure's role-based access control instead of a Shared Access Signature (SAS) if possible. If you still need to use a SAS, use a container-level access policy when creating a SAS.